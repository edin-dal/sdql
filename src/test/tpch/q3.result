<5636,1995-02-16,0,84345.6>:1
<59874,1995-01-06,0,116490>:1
<21956,1995-02-02,0,254541>:1
<28642,1995-01-19,0,87419.6>:1
<29953,1995-02-08,0,133380>:1
<55906,1994-11-27,0,46001.9>:1
<38882,1994-12-15,0,16591.2>:1
<40001,1995-02-25,0,109384>:1
<26723,1995-03-04,0,102317>:1
<19393,1995-01-15,0,38193.8>:1
<45959,1994-12-07,0,78599.1>:1
<52482,1995-02-01,0,42049.7>:1
<38784,1995-02-04,0,28136.3>:1
<21475,1995-02-21,0,119200>:1
<49537,1995-03-07,0,31834.8>:1
<59297,1994-12-10,0,15527.6>:1
<577,1994-12-19,0,46995.5>:1
<44612,1995-03-14,0,152830>:1
<450,1995-03-05,0,205447>:1
<40962,1995-02-21,0,163686>:1
<41895,1995-01-11,0,33392.3>:1
<40326,1994-12-27,0,40802.4>:1
<52640,1995-01-02,0,81063.9>:1
<30497,1995-02-07,0,208567>:1
<9248,1994-12-24,0,70712.4>:1
<1637,1995-02-08,0,243513>:1
<5347,1995-02-22,0,149754>:1
<7840,1995-01-09,0,159275>:1
<5985,1995-01-12,0,6894.69>:1
<57987,1994-12-16,0,53735.5>:1
<12706,1994-11-21,0,21677.3>:1
<27554,1995-03-11,0,119577>:1
<6022,1995-02-13,0,166150>:1
<40835,1994-11-25,0,8190.63>:1
<54787,1995-02-25,0,73044.1>:1
<46562,1995-02-13,0,56968.7>:1
<37888,1994-12-23,0,24056.2>:1
<28993,1995-03-10,0,41619.3>:1
<53634,1995-02-22,0,19348.1>:1
<45829,1995-03-13,0,48328.2>:1
<7527,1994-12-12,0,13068.3>:1
<51777,1995-02-20,0,146508>:1
<44102,1995-01-14,0,164656>:1
<59139,1995-01-30,0,67005.5>:1
<25926,1994-11-21,0,26005.1>:1
<21760,1995-01-31,0,61308.5>:1
<15138,1995-02-21,0,33617.1>:1
<35266,1995-01-11,0,153107>:1
<42081,1995-01-11,0,155730>:1
<5728,1994-12-11,0,56604>:1
<55875,1994-11-26,0,34932.2>:1
<50693,1995-01-17,0,17651.4>:1
<5031,1994-12-02,0,20587.3>:1
<59686,1995-03-09,0,13575.2>:1
<10916,1995-03-11,0,241320>:1
<47876,1994-12-26,0,75050.7>:1
<28741,1995-03-01,0,134921>:1
<46980,1995-01-18,0,66360.2>:1
<29830,1995-01-13,0,38191.8>:1
<31747,1995-02-03,0,158233>:1
<19365,1995-01-17,0,156544>:1
<26432,1994-12-31,0,35020.3>:1
<15111,1994-12-18,0,47951.1>:1
<21061,1994-12-22,0,46024.6>:1
<25669,1995-01-17,0,161663>:1
<22276,1995-01-29,0,266352>:1
<40612,1995-03-01,0,177041>:1
<45349,1995-02-12,0,94730.5>:1
<52199,1995-01-27,0,65976.1>:1
<47204,1995-03-13,0,204479>:1
<2883,1995-01-23,0,61835.6>:1
<58880,1995-01-08,0,39161.2>:1
<36999,1994-12-13,0,52966.4>:1
<47714,1995-03-11,0,267011>:1
<32128,1995-01-22,0,96799.7>:1
<22433,1995-03-04,0,149731>:1
<34727,1995-03-14,0,99510.7>:1
<43491,1995-03-09,0,140777>:1
<20453,1995-03-11,0,169158>:1
<30981,1995-02-16,0,125249>:1
<35399,1995-01-22,0,39727.5>:1
<45991,1995-02-15,0,90739.8>:1
<4227,1995-02-24,0,120474>:1
<25858,1995-01-31,0,97481.1>:1
<11073,1994-12-02,0,7289.62>:1
<16322,1994-12-20,0,92848.2>:1
<35264,1995-03-14,0,17482.4>:1
<29958,1995-01-07,0,74209.7>:1
<15462,1995-02-27,0,106315>:1
<6403,1995-02-12,0,27428.8>:1
<34724,1994-12-28,0,40788.9>:1
<28195,1994-12-21,0,4723.16>:1
<20805,1995-01-11,0,112003>:1
<46528,1994-12-07,0,93485.7>:1
<58981,1995-02-01,0,55946.6>:1
<5765,1994-12-15,0,97694.9>:1
<25027,1994-12-17,0,54115.6>:1
<33764,1995-02-11,0,160334>:1
<4707,1995-02-27,0,70091.4>:1
<33794,1995-02-23,0,116897>:1
<30182,1995-01-09,0,22516>:1
<46402,1995-01-09,0,48532.7>:1
<12961,1994-12-13,0,78376.2>:1
<19780,1995-02-05,0,87235.4>:1
<27719,1995-02-14,0,173895>:1
<56711,1995-02-07,0,38817.5>:1
<3844,1994-12-29,0,4144.59>:1
<36069,1994-12-06,0,41060.4>:1
<56295,1995-02-24,0,115111>:1
<32484,1995-02-05,0,55427.4>:1
<32901,1995-01-11,0,79789.8>:1
<30725,1994-12-29,0,167018>:1
<27361,1995-01-23,0,30722.7>:1
<10691,1995-03-14,0,84419.8>:1
<46341,1994-12-18,0,72519.6>:1
<57378,1994-12-29,0,61575.6>:1
<50945,1995-01-08,0,21476.8>:1
<56608,1994-11-19,0,27573.5>:1
<47076,1995-01-08,0,17918.1>:1
<20641,1995-02-20,0,189170>:1
<5312,1995-02-24,0,76343.6>:1
<25188,1995-01-04,0,144523>:1
<6592,1994-12-02,0,45865.3>:1
<38947,1994-11-30,0,31071.7>:1
<9696,1995-02-20,0,201502>:1
<15106,1995-01-28,0,111825>:1
<32965,1995-02-25,0,263768>:1
<17058,1995-01-02,0,50390.5>:1
<24640,1995-01-30,0,123649>:1
<13924,1994-12-20,0,3587.16>:1
<5188,1995-03-02,0,30300.8>:1
<35845,1995-01-07,0,25511.5>:1
<26848,1995-03-10,0,149819>:1
<386,1995-01-25,0,114356>:1
<47781,1995-01-29,0,79949.3>:1
<25603,1995-03-06,0,142844>:1
<59843,1995-02-14,0,195186>:1
<31140,1995-02-02,0,40285>:1
