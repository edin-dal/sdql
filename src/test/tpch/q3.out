<5985,1995-01-12,0>:<6894.69>
<40835,1994-11-25,0>:<8190.63>
<56608,1994-11-19,0>:<27573.5>
<44612,1995-03-14,0>:<38225.6>
<4707,1995-02-27,0>:<8481.22>
<28741,1995-03-01,0>:<4664.3>
<45991,1995-02-15,0>:<53100.6>
<45349,1995-02-12,0>:<52761.4>
<32128,1995-01-22,0>:<84613.8>
<35266,1995-01-11,0>:<13266.6>
<36069,1994-12-06,0>:<41060.4>
<21475,1995-02-21,0>:<46498.7>
<577,1994-12-19,0>:<27242.4>
<5765,1994-12-15,0>:<71492.4>
<59686,1995-03-09,0>:<6845.78>
<50693,1995-01-17,0>:<17651.4>
<17058,1995-01-02,0>:<1823.57>
<46528,1994-12-07,0>:<58848.2>
<41895,1995-01-11,0>:<33392.3>
<53634,1995-02-22,0>:<19348.1>
<36999,1994-12-13,0>:<52966.4>
<47076,1995-01-08,0>:<17918.1>
<1637,1995-02-08,0>:<84460>
<450,1995-03-05,0>:<61827.4>
<59297,1994-12-10,0>:<15527.6>
<35399,1995-01-22,0>:<10822.9>
<35845,1995-01-07,0>:<25511.5>
<55875,1994-11-26,0>:<34932.2>
<46341,1994-12-18,0>:<72519.6>
<20641,1995-02-20,0>:<28782.7>
<49537,1995-03-07,0>:<31834.8>
<38947,1994-11-30,0>:<31071.7>
<46980,1995-01-18,0>:<33658.6>
<56295,1995-02-24,0>:<50962.9>
<59843,1995-02-14,0>:<52400>
<9696,1995-02-20,0>:<63731.7>
<44102,1995-01-14,0>:<25677>
<30182,1995-01-09,0>:<22516>
<11073,1994-12-02,0>:<7289.62>
<7840,1995-01-09,0>:<23056.6>
<6403,1995-02-12,0>:<14196.9>
<7527,1994-12-12,0>:<13068.3>
<12706,1994-11-21,0>:<21677.3>
<29830,1995-01-13,0>:<38191.8>
<5188,1995-03-02,0>:<18613.5>
<31140,1995-02-02,0>:<40285>
<19780,1995-02-05,0>:<21486.7>
<38882,1994-12-15,0>:<16591.2>
<42081,1995-01-11,0>:<23558>
<20805,1995-01-11,0>:<35108.1>
<30981,1995-02-16,0>:<27942.2>
<5347,1995-02-22,0>:<52334.8>
<55906,1994-11-27,0>:<46001.9>
<35264,1995-03-14,0>:<17482.4>
<13924,1994-12-20,0>:<3587.16>
<29953,1995-02-08,0>:<7046.13>
<26432,1994-12-31,0>:<27761>
<30497,1995-02-07,0>:<61471.1>
<40001,1995-02-25,0>:<42922.6>
<40612,1995-03-01,0>:<29152.2>
<47204,1995-03-13,0>:<51171.1>
<32484,1995-02-05,0>:<31232.4>
<27361,1995-01-23,0>:<30722.7>
<12961,1994-12-13,0>:<44548.3>
<30725,1994-12-29,0>:<67958.3>
<31747,1995-02-03,0>:<67590.1>
<32901,1995-01-11,0>:<59565.9>
<19393,1995-01-15,0>:<38193.8>
<6592,1994-12-02,0>:<45865.3>
<29958,1995-01-07,0>:<28827.1>
<15106,1995-01-28,0>:<77070.7>
<34727,1995-03-14,0>:<47581.2>
<25669,1995-01-17,0>:<63964.1>
<38784,1995-02-04,0>:<28136.3>
<5312,1995-02-24,0>:<39128.8>
<15138,1995-02-21,0>:<33617.1>
<37888,1994-12-23,0>:<24056.2>
<26848,1995-03-10,0>:<29362.6>
<386,1995-01-25,0>:<50070.9>
<27554,1995-03-11,0>:<79206.9>
<43491,1995-03-09,0>:<17286.6>
<5636,1995-02-16,0>:<27354.7>
<25027,1994-12-17,0>:<54115.6>
<57378,1994-12-29,0>:<37798.6>
<47714,1995-03-11,0>:<19188.3>
<10691,1995-03-14,0>:<13915.6>
<25603,1995-03-06,0>:<40898.6>
<28195,1994-12-21,0>:<4723.16>
<32965,1995-02-25,0>:<62114.6>
<5728,1994-12-11,0>:<56604>
<40326,1994-12-27,0>:<40802.4>
<24640,1995-01-30,0>:<55569>
<27719,1995-02-14,0>:<20813.1>
<59139,1995-01-30,0>:<26881.4>
<26723,1995-03-04,0>:<11252.8>
<2883,1995-01-23,0>:<61835.6>
<6022,1995-02-13,0>:<32702.6>
<22276,1995-01-29,0>:<27280.4>
<34724,1994-12-28,0>:<40788.9>
<47781,1995-01-29,0>:<31872.1>
<4227,1995-02-24,0>:<27717.9>
<28993,1995-03-10,0>:<41619.3>
<33764,1995-02-11,0>:<79571.7>
<45959,1994-12-07,0>:<78599.1>
<59874,1995-01-06,0>:<19694.9>
<21061,1994-12-22,0>:<46024.6>
<40962,1995-02-21,0>:<24581.1>
<47876,1994-12-26,0>:<54892.5>
<46562,1995-02-13,0>:<1591.98>
<5031,1994-12-02,0>:<20587.3>
<25188,1995-01-04,0>:<37980>
<46402,1995-01-09,0>:<6660>
<52199,1995-01-27,0>:<48895.6>
<52640,1995-01-02,0>:<19415.8>
<58880,1995-01-08,0>:<16710.2>
<25926,1994-11-21,0>:<26005.1>
<52482,1995-02-01,0>:<1478.43>
<25858,1995-01-31,0>:<11990.3>
<33794,1995-02-23,0>:<33611.5>
<21956,1995-02-02,0>:<68243.8>
<15462,1995-02-27,0>:<33682.4>
<21760,1995-01-31,0>:<61308.5>
<15111,1994-12-18,0>:<47951.1>
<19365,1995-01-17,0>:<51515.4>
<22433,1995-03-04,0>:<21856.1>
<3844,1994-12-29,0>:<4144.59>
<51777,1995-02-20,0>:<15649.1>
<56711,1995-02-07,0>:<17892.1>
<9248,1994-12-24,0>:<31849.1>
<57987,1994-12-16,0>:<53735.5>
<16322,1994-12-20,0>:<13637>
<20453,1995-03-11,0>:<47266.5>
<45829,1995-03-13,0>:<48328.2>
<10916,1995-03-11,0>:<67307>
<54787,1995-02-25,0>:<9727.47>
<58981,1995-02-01,0>:<16887.4>
<50945,1995-01-08,0>:<21476.8>
<28642,1995-01-19,0>:<27980>
